tests/integration/extenders/ThemeTest.php

```diff
@@ -104,4 +104,32 @@ public function theme_extender_override_file_source_works_by_failing_when_necess
         $this->assertStringContainsString('Less_Exception_Compiler', $response->getBody()->getContents());
         $this->assertEquals(500, $response->getStatusCode());
     }
+
+    /**
+     * @test
+     */
+    public function theme_extender_can_add_custom_function()
+    {
+        $this->extend(
+            (new Extend\Frontend('forum'))
+                ->css(__DIR__.'/../../fixtures/less/custom_function.less'),
+            (new Extend\Theme)
+                ->addCustomLessFunction('is-flarum', function ($text) {
+                    return strtolower($text) === 'flarum' ? 'true' : 100;
+                })
+                ->addCustomLessFunction('is-gt', function ($a, $b) {
+                    return $a > $b === 'flarum' ? true : false;
```

Comment:
Why is `=== 'flarum' in here?

Reply:
Also shouldn't the returned values be strings, not raw bools?

Reply:
1. I have no idea.
2. No, we should transform them appropriately, hence the test.

Reply:
```suggestion
                    return $a > $b;
```